jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -27,6 +27,7 @@ LIT_MAGIC_STRING_FIRST_STRING_WITH_SIZE (1, LIT_MAGIC_STRING_NEW_LINE_CHAR)
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_NEW_LINE_CHAR, "\n")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SPACE_CHAR, " ")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_DOUBLE_QUOTE_CHAR,"\"")
+LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SINGLE_QUOTE_CHAR,"'")
```

Comment:
Tunes out introduce a format function to compose error message can make things easier:

```c
ecma_format_ecma_string ("Cannot read property '%' of %", prop_str_p, name_str_p)
```

- No need to involve magic string.
- The format string can be easily found with `grep`.
- Also, with this util [parser_parse_script](https://github.com/jerryscript-project/jerryscript/blob/master/jerry-core/parser/js/js-parser.c#L2391) can be simplified.

@zherczeg